logger.go

```diff
@@ -9,15 +9,16 @@ import (
 	"github.com/gobuffalo/pop/v5/logging"
 )
 
-type logger func(lvl logging.Level, s string, args ...interface{})
+type Logger func(lvl logging.Level, s string, args ...interface{})
 
 // Debug mode, to toggle verbose log traces
 var Debug = false
 
 // Color mode, to toggle colored logs
 var Color = true
 
-var log logger
+var log Logger
+var standardLogger *stdlog.Logger
```

Comment:
The name `defaultStdLogger` actually means something like "default log exporter to stdout". (yeah, not a perfect name) The name of this new var also does not represent the purpose of the var enough. (`SetStandardLogger()` also the same.)

Also, the type `*log.Logger` is also not flexible.